.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Please don't add your IDE files (the entire `./idea` folder) to the project, *especially* in an unrelated commit.

Feel free to add a `/.idea/` line to the top-level .gitignore.